packages/core/contracts/financial-templates/common/FeePayer.sol

```diff
@@ -148,6 +145,27 @@ abstract contract FeePayer is AdministrateeInterface, Testable, Lockable {
         return totalPaid;
     }
 
+    function getOutstandingRegularFees(uint256 time)
```

Comment:
this view method is pulled out & re-used in the `payRegularFees` and `_getPendingRegularFeeAdjustedCollateral` functions.